#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_paths_in_config_file.dpatch by  <lars@erker.wg>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change the default paths in the ezmlmwebrc config file

@DPATCH@

--- ezmlm-web-3.2/examples/ezmlmwebrc.dist.orig	2006-05-04 05:23:54.000000000 +0200
+++ ezmlm-web-3.2/examples/ezmlmwebrc.dist	2006-05-04 05:24:06.000000000 +0200
@@ -26,11 +26,11 @@
 
 # Where are the language files
 # usually something like /usr/local/share/ezmlm-web/lang
-$LANGUAGE_DIR = "/usr/local/share/ezmlm-web/lang";
+$LANGUAGE_DIR = "/usr/share/ezmlm-web/lang";
 
 # Where are the template files
 # usually something like /usr/local/share/ezmlm-web/template
-$TEMPLATE_DIR = "/usr/local/share/ezmlm-web/template";
+$TEMPLATE_DIR = "/usr/share/ezmlm-web/template";
 
 # Safe list deletion? 
 # 0 = move list to $LIST_DIR/_deleted_lists -> recoverable :)
